test/integration/dra/dra_test.go

```diff
@@ -574,6 +593,44 @@ func testPrioritizedList(tCtx ktesting.TContext, enabled bool) {
 	}).WithTimeout(10 * time.Second).WithPolling(time.Second).Should(schedulingAttempted)
 }
 
+func testExtendedResource(tCtx ktesting.TContext, enabled bool) {
```

Comment:
Please cover the following scenarios:
- one container, one extended resource
- one container, three extended resources
- three containers, one extended resource each
- three containers, three extended resources

This is relevant for the `listMap` handling of `RequestMapping`.